app/src/Menus/Menu.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+
+import * as apiClient from "../apiClient";
+import WhiteBorder from "../images/border-style-white.png";
+
+import CourseType from "./CourseType";
+import FilterSection from "./FilterSection";
+import styles from "./styles.module.scss";
+
+function Menu({ type }) {
+  const [info, setInfo] = React.useState([]);
+  const [fullMenu, setFullMenu] = React.useState({});
+  const courses = info[`${type.toLowerCase()}_courses`];
```

Comment:
Looks like this should be evaluated following `loadInfo` since there's no data yet.